src/kit/RadioGroup.tsx

```diff
@@ -18,7 +18,7 @@ interface Props {
   onChange?: (id: string | number) => void;
```

Comment:
chore: after looking through the paired PR, i realized this interface could be made into a generic, something like `interface Props<T extends string | number>`. that way `defaultValue`, `onChange`, and `value` can all be constrained to the same type `T`

Reply:
good call, will do!